### PR TITLE
Feature/npm packages update

### PR DESCRIPTION
**Pull request by dan-gamble**
This is reliant on #153 and can't be merged until that has.

### CODE REVIEW

File: {{cookiecutter.repo_name}}/package.json [hunk 1]
```diff
@@ -198,7 +198,7 @@
         1
       ]
     }
-  },
+  }
```
Comment (lewiscollard) [inline {{cookiecutter.repo_name}}/package.json]: Is this missing a comma?

Comment (dan-gamble) [reply {{cookiecutter.repo_name}}/package.json]: Yep!

File: {{cookiecutter.repo_name}}/package.json [hunk 2]
```diff
@@ -444,7 +444,6 @@
       "at-rule-name-case": "lower",
       "at-rule-name-space-after": "always",
       "at-rule-no-vendor-prefix": true,
-      "stylelint-disable-reason": "always-before",
```
Comment (lewiscollard) [inline {{cookiecutter.repo_name}}/package.json:402]: I personally rather liked this rule. Is there a newer alternative to it, or a reason it is deprecated?

Comment (dan-gamble) [reply {{cookiecutter.repo_name}}/package.json:402]: > The following rules did not work well.
stylelint-disable-reason could not enforce providing a reason

Is all they seem to say. I was hit or miss on it personally.

Comment (danielsamuels) [reply {{cookiecutter.repo_name}}/package.json:402]: I liked it, but there's nothing stopping you from continuing adding a reason above regardless of the enforcement.

Comment (dan-gamble) [reply {{cookiecutter.repo_name}}/package.json:402]: Yea, and if you still wanted to enforce it, stylelint rules are pretty easy to write. Could just take their source code and create a plugin from it

Comment (lewiscollard) [reply {{cookiecutter.repo_name}}/package.json:402]: Thinking about it, I see their logic completely (they they can only enforce that _there is a comment above it_, not an actual _reason_). I plan to always add a reason either way. :)

**Review by lewiscollard — APPROVED** [review]

**Pull request merged**